eclipse-scout-core/src/form/fields/FormField.ts

```diff
@@ -847,10 +847,7 @@ export class FormField extends Widget implements FormFieldModel {
   }
 
   protected _setGridDataHints(gridData: GridData) {
-    if (!gridData) {
-      gridData = new GridData();
-    }
-    this._setProperty('gridDataHints', GridData.ensure(gridData));
+    this._setProperty('gridDataHints', GridData.ensure(gridData || new GridData()));
```

Comment:
It ensures the passed object has the GridData type, unless it is null or undefined. We use the same pattern for dates.ensure, numbers.ensure, LookupCall.ensure, DateFormat.ensure. I think this is correct because otherwise we would lose the null state which may be important. For arrays we explicitly don't want the null state. But I agree it is a little bit confusing...